code/go/0chain.net/chaincore/chain/state.go

```diff
@@ -573,6 +581,9 @@ func (c *Chain) mintAmount(sctx bcstate.StateContextI, toClient datastore.Key, a
 func (c *Chain) validateNonce(sctx bcstate.StateContextI, fromClient datastore.Key, txnNonce int64) error {
 	s, err := c.GetStateById(sctx.GetState(), fromClient)
 	if !isValid(err) {
+		logging.Logger.Error("validate nonce - error",
```

Comment:
We don't need to log the error here, the only possible error here is 'node not found', which means the miner/sharder is not synced yet, and we will see this error in upper caller's logs.

Reply:
removed